superbench/benchmarks/micro_benchmarks/cublas_function/cublas_helper.h

```diff
@@ -50,9 +50,11 @@ void cuda_free(cublasHandle_t *cublas_handle);
  * @param  a                input matrixA
```

Comment:
why change this for correctness?

Reply:
because I want to move alpha and beta to be function auguments, this is because the matrix multilply in the cublas functions is C = alpha*A*B + beta*C, if beta is 1, in each step, the C will change and bring difficulty to calculate the ground truth on CPU, so if the correctness check is enabled, I force set beta to be 0